test/controllers/update/update.review.statement.controller.spec.ts

```diff
@@ -141,29 +152,53 @@ describe("Update review overseas entity information controller tests", () => {
   });
 
   describe("POST tests", () => {
-    test(`redirect to ${PAYMENT_LINK_JOURNEY}, with transaction and OE id`, async () => {
+    test(`redirect to ${PAYMENT_LINK_JOURNEY}, with transaction and OE id (and no_change_review_statement undefined)`, async () => {
       mockIsActiveFeature.mockReturnValueOnce(true);
       mockGetApplicationData.mockReturnValueOnce(APPLICATION_DATA_UPDATE_BO_MOCK);
       mockPaymentsSession.mockReturnValueOnce(PAYMENT_LINK_JOURNEY);
-      await request(app).post(UPDATE_REVIEW_STATEMENT_URL);
-
-      // assertions dependent on UAR-587
+      const resp = await request(app).post(UPDATE_REVIEW_STATEMENT_URL);
 
-      // expect(resp.status).toEqual(302);
-      // expect(resp.header.location).toEqual(PAYMENT_LINK_JOURNEY);
+      expect(resp.status).toEqual(302);
+      expect(resp.header.location).toEqual(PAYMENT_LINK_JOURNEY);
     });
 
     test(`redirect to ${PAYMENT_LINK_JOURNEY}, if Save and Resume not enabled`, async () => {
       mockIsActiveFeature.mockReturnValueOnce(false);
       const mockData = { ...APPLICATION_DATA_UPDATE_BO_MOCK, [Transactionkey]: "", [OverseasEntityKey]: "" };
       mockGetApplicationData.mockReturnValueOnce(mockData);
       mockPaymentsSession.mockReturnValueOnce(PAYMENT_LINK_JOURNEY);
-      await request(app).post(UPDATE_REVIEW_STATEMENT_URL);
+      const resp = await request(app).post(UPDATE_REVIEW_STATEMENT_URL);
+
+      expect(resp.status).toEqual(302);
+      expect(resp.header.location).toEqual(PAYMENT_LINK_JOURNEY);
+    });
+
+    test(`should redirect to UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_URL when noChangeReviewStatement is '0'`, async () => {
+      const mockData = {
+        ...APPLICATION_DATA_UPDATE_BO_MOCK,
+        [Transactionkey]: "",
+        [OverseasEntityKey]: "",
+      };
+      mockGetApplicationData.mockReturnValueOnce(mockData);
```

Comment:
Can be replaced with:

`mockGetApplicationData.mockReturnValueOnce(APPLICATION_DATA_UPDATE_BO_MOCK);`